git_machete/tests/functional/test_machete.py

```diff
@@ -54,6 +54,10 @@ def mock_fetch_ref(cls: Any, remote: str, ref: str) -> None:
     git.checkout(branch)
 
 
+def fake_get_github_token() -> Optional[str]:
```

Comment:
Hmm there's already `mock__get_github_token`... pls keep the naming convention consistent + make it clear from the name how the two methods differ